src/main/java/frc/robot/subsystems/chassis/Chassis.java

```diff
@@ -76,7 +76,9 @@ public ChassisSpeeds getRealChassisSpeed() {
   }
 
   public void driveRobotRelative(ChassisSpeeds speeds) {
-    SwerveModuleState[] states = kinematics.toSwerveModuleStates(speeds);
+    ChassisSpeeds setSpeeds = speeds;
+    setSpeeds = ChassisSpeeds.discretize(setSpeeds, 0.02);
+    SwerveModuleState[] states = kinematics.toSwerveModuleStates(setSpeeds);
```

Comment:
```suggestion
    ChassisSpeeds discretizedSpeeds = ChassisSpeeds.discretize(speeds, 0.02);
    SwerveModuleState[] states = kinematics.toSwerveModuleStates(discretizedSpeeds);
```

Reply:
Hmm, no idea why I did it that way haha. Thanks